### PR TITLE
Ship a bundled version of artsy-passport as a node module

### PR DESCRIPTION
**Pull request by orta**
I've bumped the version number because whilst there are zero behaviour changes, shipping JS instead of CoffeeScript is a breaking change IMO.

This had some trickiness due to https://github.com/substack/coffeeify/issues/1 - you can see a full (kinda explicit) writeup here: https://github.com/ELLIOTTCABLE/Paws.js/commit/75fb58037914899755bbcd241e47fde060b237dc

But the general gist is that I needed to add `.coffee` to any relative require in order to make it compile. Will add a few notes inline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,26 @@
...
+## Breaking changes in 2.0
+
+* The app is now shipped as a single JS file.
+
 ## Setup
 
 #### Make sure you first mount session, body parser, and start [artsy-xapp](https://github.com/artsy/artsy-xapp).
 
-````coffeescript
+```coffee
```
Comment (orta) [inline README.md]: wasn't showing up in my editor 🍡

File: package.json [hunk 2]
```diff
@@ -45,22 +46,24 @@
...
     "underscore": "*",
-    "backbone": "*",
-    "sharify": "*",
-    "backbone-super-sync": "*",
-    "browserify": "*",
-    "coffeeify": "*",
-    "artsy-xapp": "*",
-    "zombie": "*",
-    "rewire": "2.2.0"
+    "zombie": "*"
```
Comment (orta) [inline package.json]: `brfs` is new to the package, and `coffee-script` is now in dev-deps. Somewhere in the browserify + coffeeify def tree `brfs` is used but not included in their dependencies. It's re-ordered to be alphabetical.

**Comment by mzikherman** [comment]
👍

**Comment by craigspaeth** [comment]
Thank you! Now who's going to run it through [decaffinate](https://github.com/decaffeinate/decaffeinate) 😬

**Pull request merged**